client/blocks/comments/post-comment.jsx

```diff
@@ -106,8 +108,9 @@ class PostComment extends PureComponent {
 		this.setState( { showReplies: ! this.state.showReplies } );
 	};
 
-	handleReply = () => {
-		if ( ! this.props.isLoggedIn ) {
+	onLikeToggle = () => {
```

Comment:
I notice a couple places where we define an `onLikeToggle` in this PR. In reading these, I was initially confused as I saw no logic around opening the login modal (or setting the logged in action), but see things related to opening other login tabs if the login modal isn't enabled. I later saw we handle login modal down the chain where we determine where to call this `onLikeToggle`. I wonder if it makes sense to consolidate some of this logic together, or if theres a reason they need to be more separated.

Reply:
I noticed the same and was trying to make as little change as possible here to be honest. I think we could use another PR to refactor and consolidate the comment and post likes - I didn't want to tackle that here.

Reply:
Just to clarify my comment above! The like code for the comment and post are currently handled separately. I think we should be able to refactor them to consolidate how the underlying like behaviour works but I felt it was beyond scope of this PR and even project. I think it would be worth doing at some point though.